### ISSUE TITLE
Examples with IO

### ISSUE THREAD

**Issue by casperisfine** [ivoanjo/gvl-tracing#5]
It would be nice if not all examples were very CPU heavy.

One thing that is often hard to teach about the GVL is when things can actually happen in parallel or not, as such it would be nice to have some examples that do some IOs to show how it release the GVL for other threads.

It would also make for good screenshot in the README so that people don't start to think threads are entirely worthless.

**Comment by ivoanjo** [comment]
Yeap, fully agreed. I initially intended to include it in the blog post, but ended up "scaling back" the number of examples to hit the first release ;)

**Comment by ivoanjo** [comment]
Yeap -- having more examples in the README would definitely be welcome
